db/data/regions/BE.yml

```diff
@@ -24,5 +24,9 @@ languages:
 format:
   edit: "{country}_{firstName}{lastName}_{company}_{address1}_{address2}_{zip}{city}_{phone}"
   show: "{firstName} {lastName}_{company}_{address1}_{address2}_{zip} {city}_{country}_{phone}"
+additional_address_fields:
+  address1:
+    - key: street
+    - key: building_number
```

Comment:
Discussed updating `building_number` -> `buildingNumber` and including a key to indicate whether the field is required.